tmva/sofie_parsers/src/RModelParser_ONNX.cxx

```diff
@@ -37,18 +38,27 @@ std::unique_ptr<ROperator> make_ROperator_Add(const onnx::NodeProto& nodeproto,
          else
             assert(it->second == input_type);
       } else {
-         throw std::runtime_error("TMVA::SOFIE ONNX Parser Add op has input tensor" + input_name + " but its type is not yet registered");
+         // check if input tensor is an initialized tensor
+         bool isInitializer = false;
+         for (int i=0; i < graphproto.initializer_size(); i++){
```

Comment:
please change index i to j to fix the warning:
```
for (int j=0; j < graphproto.initializer_size(); j++){
if (input_name == graphproto.initializer(j).name())
.....
```
..

Reply:
Done!

Reply:
Thank you !